src/js/components/FormField/FormField.js

```diff
@@ -11,9 +11,43 @@ import { withTheme } from '../hocs';
 
 import doc from './doc';
 
+// empirically determined via interaction with story, post IE11 '1' is enough.
+const FOCUS_DELAY = 20;
+
 class FormField extends Component {
   state = {};
 
+  componentWillUnmount() {
+    clearTimeout(this.focusTimer);
+  }
+
+  onClick = () => {
+    // set focus on focusable descendant
+    const container = findDOMNode(this.childContainerRef);
+    const element = getFirstFocusableDescendant(container);
+    if (element) {
+      element.focus();
+    }
+  }
+
+  onFocus = () => {
```

Comment:
have you considered using withFocus hoc ?

Reply:
I think the idea here is that the FormField never receives focus but just listens for onClick and passes the focus on to the appropriate underlying element. But, just to check, I tried using withFocus but it didn't work. I didn't delve too far into why.

Reply:
but you are setting `focus` to the state right ? I would be curious to understand why withFocus did not work for you.